src/main/java/core/mate/academy/service/TruckProducer.java

```diff
@@ -0,0 +1,16 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Truck;
+import java.util.List;
+
+public class TruckProducer implements MachineProducer<Truck> {
+    private Truck volvoTruck = new Truck();
+    private Truck mercedesTruck = new Truck();
+
+    private List<Truck> truckList = List.of(volvoTruck, mercedesTruck);
```

Comment:
`to produce` means to create new objects each time when we are trying to get them, so please move this logic to the `get` method
```suggestion
```